tests/model_registry/conftest.py

```diff
@@ -298,8 +303,44 @@ def model_registry_instance_rest_endpoint(
 
 
 @pytest.fixture(scope="class")
-def generated_schema(model_registry_instance_rest_endpoint: str) -> Any:
-    return schemathesis.from_uri(
-        uri="https://raw.githubusercontent.com/kubeflow/model-registry/main/api/openapi/model-registry.yaml",
-        base_url=f"https://{model_registry_instance_rest_endpoint}/",
+def generated_schema(model_registry_instance_rest_endpoint: str) -> BaseOpenAPISchema:
+    schema = schemathesis.openapi.from_url(
+        url="https://raw.githubusercontent.com/kubeflow/model-registry/main/api/openapi/model-registry.yaml"
     )
+    schema.configure(base_url=f"https://{model_registry_instance_rest_endpoint}/")
+    return schema
+
+
+# Used when testing local changes
+# @pytest.fixture(scope="class")
+# def generated_schema_local(model_registry_instance_rest_endpoint):
+#     schema = schemathesis.openapi.from_path(
+#         path="utilities/manifests/model-registry-api.yaml",
```

Comment:
please set path in py_config so when a user wants to overwrite they can us tc=<param name>:<path>

Reply:
Not sure to which scenario this would apply, this needs to be manually enabled and pointing to an updated openapi spec, I don't think it makes sense to have it exposed as a config variable.

Reply:
Removed